src/Sulu/Bundle/AdminBundle/Resources/public/js/components/login/main.js

```diff
@@ -459,7 +461,8 @@ define([], function() {
          * Handle click on login-button in login-frame
          */
         loginButtonClickHandler: function() {
-            this.sandbox.dom.submit(this.dom.$loginForm);
+            // click hidden submit-button to provide password-saving in internet-explorer
+            $('#' + constants.loginSubmitButtonId).click();
```

Comment:
Shouldn't it be possible to just use `submit`? Would be a bit more straight forward than emulating a click on a hidden element.